gradio/components.py

```diff
@@ -2500,7 +2501,7 @@ def __process_markdown(cls, data: List[List[Any]], datatype: List[str]):
             return data
 
         if cls.markdown_parser is None:
-            cls.markdown_parser = MarkdownIt().enable("table")
+            cls.markdown_parser = MarkdownIt().use(dollarmath_plugin).enable("table")
```

Comment:
I'm not sure we want to support math syntax in tables, we certainly want to test this thoroughly as `display mode` will probably cause problems. Inline mode is probably fine.

Reply:
I don't believe mathtext supports display mode, only inline latex.

Reply:
should i then add the `utils.tex2svg` renderer here?

Reply:
oh good catch, I'll do that